src/net.cpp

```diff
@@ -1705,31 +1702,37 @@ void static Discover(boost::thread_group& threadGroup)
 void StartNode(boost::thread_group& threadGroup, CScheduler& scheduler)
 {
     uiInterface.InitMessage(_("Loading addresses..."));
-    // Load addresses for peers.dat
+    // Load addresses from peers.dat
     int64_t nStart = GetTimeMillis();
     {
         CAddrDB adb;
         if (!adb.Read(addrman))
             LogPrintf("Invalid or missing peers.dat; recreating\n");
+        else
+            LogPrintf("Loaded %i addresses from peers.dat  %dms\n", addrman.size(), GetTimeMillis() - nStart);
     }
 
-    //try to read stored banlist
+    uiInterface.InitMessage(_("Loading banned addresses..."));
```

Comment:
Hmm.... it could be `_("Loading banned addresses/subnets...")` but if we be particular about this: a subnet defines an amount of "addresses". But i don't care about that.